contracts/tokenization/StakedToken.sol

```diff
@@ -3,10 +3,7 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "./IStakedToken.sol";
-
-interface IStakeManager {
-    function getStake(uint32 stakerId) external view returns (uint256);
-}
+import "../Core/interface/IStakeManager.sol";
```

Comment:
Hey @adi44 
Was there any warning to use this instead?
I had kept that like this, as sToken is going to be deployed multiple times.
so if we import all, bytecode increases needlessly, which user has to bear cost for

Reply:
So, there was warning related to same variable name, slither detected IstakeManager to be present in stakedtoken contract and in interface directory. I think in that case we could just ignore this warning, if it increases the cost.

Reply:
cool

Reply:
there is no significant increase in the stake Gas Consumption.